### PR TITLE
chore: publish to JetBrains plugin portal

### PR DESCRIPTION
**Pull request by pavel-github**
This PR adds a functionality to deploy plugin into JB portal.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,13 +21,23 @@ before_deploy:
...
+      - distribution/target/teamcity-snyk-security-plugin.zip.sha256
+    skip_cleanup: true
+    on:
+      repo: snyk/teamcity-snyk-security-plugin
+      tags: true
+  # JetBrains plugin portal
+  - provider: script
+    script:
+      echo "Deploying the plugin to JetBrains Portal..." &&
+      curl -k -i --request POST -F userName="$JETBRAINS_USERNAME" -F password="$JETBRAINS_PASSWORD" -F pluginId=12227 -F file=@distribution/target/teamcity-snyk-security-plugin.zip https://plugins.jetbrains.com/plugin/uploadPlugin
```
Comment (adrukh) [inline .travis.yml]: I'd rather drop the `-k` so we get the full benefit of SSL hostname matching.

Comment (pavel-github) [reply .travis.yml]: good catch.
fixed with c4ec1b8

File: pom.xml [hunk 2]
```diff
@@ -132,7 +132,7 @@
   <repositories>
     <repository>
       <id>jetbrains-all</id>
-      <url>https://download.jetbrains.com/teamcity-repository</url>
+      <url>http://download.jetbrains.com/teamcity-repository</url>
```
Comment (adrukh) [inline pom.xml]: @pavel-github please revert this single line :)

Comment (pavel-github) [reply pom.xml]: you're right :), after rebasing feature branch I don't need to revert this commit.

**Review by adrukh — APPROVED** [review]

**Pull request merged**